src/com/lftechnology/java/training/srijan/basics/coreapp/service/EmployeeServiceImpl.java

```diff
@@ -0,0 +1,118 @@
+
+package com.lftechnology.java.training.srijan.basics.coreapp.service;
+
+import com.lftechnology.java.training.srijan.basics.coreapp.dao.DaoFactory;
+import com.lftechnology.java.training.srijan.basics.coreapp.dao.EmployeeDaoImpl;
+import com.lftechnology.java.training.srijan.basics.coreapp.domain.Employee;
+import com.lftechnology.java.training.srijan.basics.coreapp.domain.SearchEmployee;
+import com.lftechnology.java.training.srijan.basics.coreapp.domain.UserRole;
+
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class EmployeeServiceImpl implements EmployeeService {
+
+	private String nullUserNameMsg;
+	private String nullPasswordMsg;
+	private static final Logger LOGGER =
+		Logger.getLogger(EmployeeServiceImpl.class.getName());
+
+	public EmployeeServiceImpl() {
+		this.nullUserNameMsg = "Username cannot be Empty";
+		this.nullPasswordMsg = "Password cannot be Empty";
+	}
+
+	@Override
+	public void addEmployee(Employee e) {
+		if (!"".equals(e.getUserName()) && !"".equals(e.getPassword()) &&
+			!"".equals(e.getFullName()) && !"".equals(e.getDepartment())) {
+			EmployeeDaoImpl addEmployeeToDb = DaoFactory.getEmployeeDao();
+			addEmployeeToDb.addEmployee(e);
+		}
+		else {
+			LOGGER.log(Level.INFO, "please fill up all information");
+		}
+	}
+
+	@Override
+	public Employee findById(int idToEdit) {
+
+		EmployeeDaoImpl updateEmployeeDetails = DaoFactory.getEmployeeDao();
+		return updateEmployeeDetails.findById(idToEdit);
+	}
+
+	public UserRole save(Employee e) {
+		UserRole role;
+		if (!"".equals(e.getUserName()) && !"".equals(e.getPassword())) {
+			EmployeeDaoImpl loginValidator = DaoFactory.getEmployeeDao();
+			System.out.println();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)